pages/data-providers/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { useRedirect } from '../_app/hooks'
+
+import { withGlobalStore } from 'store'
+import { Card } from 'design'
+
+const AccessError = () => (
+  <Card>
+    <h1>
+      Access error{' '}
+      <span role="img" aria-label="sadly">
+        😞
+      </span>
+    </h1>
+    <p>There is no repository or journal you have access to.</p>
+  </Card>
+)
+
+// It's not a real list. It's a redirect at the moment
+const DataProviders = ({ store }) => {
+  const { dataProviders } = store
+  const id = dataProviders && dataProviders[0].id
+
+  // TODO: Find a way to make this conditional
+  useRedirect('[data-provider-id]', `${id}`)
```

Comment:
Maybe explicit `toString()` will let us know when something goes wrong, i.e. got `null` or `undefined`

```suggestion
  // TODO: Find a way to make this conditional
  useRedirect('[data-provider-id]', id.toString())
```

---
_TODO_ is still a question.

Reply:
@Joozty any ideas about conditional hooks? It will fail if an error happens and will redirect to `undefined`.

Reply:
Conditional hooks are bad practice there is even an eslint rule for it. I saw SWR has some sort of conditional hooks. Maybe you can have a look there or google something. Sorry, I am not much help here. It's new for me too.

Reply:
I see the only possibility to pass there a function that either returns undefined if the hook is not needed or actual values if it is. 

Also, I can check first parameter to be truthy if hook is needed.